jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/signin.js

```diff
@@ -40,18 +40,11 @@ $(function () {
         var footerContent = '<input  type="submit" id="signin-submit-button" value="' + $labelSignin + '" class="btn btn-primary" name="commit"/>';
 
         var submitDialog = function (e) {
-            if (e.keyCode == enterCode) {
-                //if focus on username then select password field
-                if ($(e.target).is('#userName')) {
+            if((e.keyCode || e.charCode) == enterCode) {
```

Comment:
Remember that jquery has the desired property "event.which". 
Instead custom global function will be better to use it's.